__tests__/crowdfundings/payPledge.test.js

```diff
@@ -205,18 +216,17 @@ test('failed payments on MONTHLY_ABO pledge with STRIPE lead to disabled members
     pledgeId
   }, pgDatabase())
 
-  // triggered from stripe subscription
+  // triggered from stripe subscription, this happens after X tries to charge
   await cancelSubscription({
     pledgeId,
     status: 'canceled',
-    atPeriodEnd: true
+    atPeriodEnd: false
   }, pgDatabase())
 
   const membership = await pgDatabase().public.memberships.findOne({ pledgeId })
-  t.equal(membership.active, false, 'membership is not active')
+  t.equal(membership.active, false, 'membership is not active anymore')
```

Comment:
I'm not sure if you test this somewhere else already, otherwise you chould check if the user lost the `member` role

Reply:
https://github.com/orbiting/republik-backend/pull/36#discussion_r160418864